stat/distuv/f.go

```diff
@@ -109,9 +109,6 @@ func (f F) Skewness() float64 {
 //
 // StdDev returns NaN if the D2 parameter is less than or equal to 4.
 func (f F) StdDev() float64 {
-	if f.D2 <= 4 {
-		return math.NaN()
-	}
```

Comment:
I think this is likely there to avoid to the SQRTSD call. Removing these lines actually significantly increases the complexity of the generated code, so can you revert this.

Reply:
But this this only matters if D2 <= 4. Do you think that in actual usage Variance will be called often for cases where D2 <= 4?

Reply:
My point was that the generated code is more complex and longer for no real gain.

Reply:
Done.